### PR TITLE
Run tests in PR

### PR DESCRIPTION
**Pull request by vlad-yakovlev**
Add GH action to run tests in PRs

Make sure that you:

- [X] Typedoc added for new methods and updated for changed
- [X] Tests added for new methods and updated for changed
- [X] New methods added to `src/index.ts`
- [X] New methods added to `mapping.md`

### CODE REVIEW

File: .github/workflow/pr.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 10
+          - 12
```
Comment (TkDodo) [inline .github/workflow/pr.yml:14]: i wouldn't run against 10 or 12. They are both end of life already.

**Comment by TkDodo** [comment]
shouldn't the action already run on this PR? Any ideas what I'd need to do setup to make that work?

**Comment by TkDodo** [comment]
aah, the directory needs to be `.github/workflows` instead of `.github/workflow` !

**Comment by TkDodo** [comment]
hm, its still not running. it could be that it needs to be merged to master first in order to be picked up?

**Comment by vlad-yakovlev** [comment]
Yep, my mistake, `.github/workflows` should be used ) And it needs to be merged for run

**Comment by TkDodo** [comment]
okay, it's merged. let's try on another PR  if its being picked up

**Comment by vlad-yakovlev** [comment]
#187 - tested & compiled successfully 🎉

**Comment by TkDodo** [comment]
awesome 🚀

**Pull request merged**